requirements/test.txt

```diff
@@ -37,38 +37,36 @@ filelock==3.0.12          # via tox, virtualenv
 future==0.18.2            # via -r requirements/base.txt, pyjwkest
 gunicorn==19.1.1          # via -r requirements/base.txt
 idna==2.8                 # via -r requirements/base.txt, requests
-importlib-metadata==1.7.0  # via inflect, pluggy, pytest, tox, virtualenv
-importlib-resources==3.0.0  # via virtualenv
-inflect==3.0.2            # via jinja2-pluralize
+inflect==4.1.0            # via jinja2-pluralize
+iniconfig==1.0.1          # via pytest
 itypes==1.2.0             # via -r requirements/base.txt, coreapi
 jinja2-pluralize==0.3.0   # via diff-cover
 jinja2==2.11.2            # via -r requirements/base.txt, code-annotations, coreschema, diff-cover, jinja2-pluralize
-lazy-object-proxy==1.5.0  # via astroid
+lazy-object-proxy==1.5.1  # via astroid
 markupsafe==1.1.1         # via -r requirements/base.txt, jinja2
-mock==3.0.5               # via -r requirements/test.in
+mock==4.0.2               # via -r requirements/test.in
```

Comment:
Since pyupgrade switched the code to use `unittest.mock` instead, I don't think this is needed anymore.